FileIO/Legacy/MeshIO.cpp

```diff
@@ -103,7 +105,9 @@ MeshLib::Mesh* MeshIO::loadMeshFromFile(const std::string& file_name)
 				for (unsigned i = 0; i < nElements; ++i)
 				{
 					getline(in, line_string);
-					elements.push_back(readElement(line_string, nodes));
+					std::stringstream ss(line_string);
+					materials.push_back(readMaterialID(ss));
+					elements.push_back(readElement(ss, nodes));
```

Comment:
I did not change the already implemented behaviour. What do you think? Opinions of other developers are also wellcome. If we decide to change the behaviour should we do this in a separate PR?